### ISSUE TITLE
create/send message

### ISSUE THREAD

**Issue by Padle** [directus/directus-sdk-javascript#34]
**Hello there!**

In my current project I want to send messages with hints and problem messages to some users and user groups.

## Create/Send Message
This Feature is supported in the PHP SDK: [Create/Send Message](https://github.com/directus/api-docs/blob/1.1/SDKs/php-sdk.md#createsend-messages). So I did some research.

I looked at the API code and found out that a POST request to `messages/rows/` is possible.
In [api/api.php](https://github.com/directus/directus/blob/c16e230e9ffea2b5426a19c22764451493818e5b/api/api.php#L485) is a route set up for post and also a function in [api/routes/A1/Messages.php](https://github.com/directus/directus/blob/c16e230e9ffea2b5426a19c22764451493818e5b/api/routes/A1/Messages.php#L161).

Unfortunately, sending messages via the API is currently not mentioned in the API docs or just forgotten.

The data send with POST to `messages/rows/` could look like this:
```JSON
   {
        "from": 7,
        "recipients": "0_7, 0_5, 1_1",
        "subject": "The Subject of the Message",
        "message": "The Message for the users"
    }
```
First the explode() function breaks the string of recipients into an array
```PHP
   $recipients = explode(',', $requestPayload['recipients']); 
```
and then the system checks whether the id belongs to a group or user.
```PHP
   foreach ($recipients as $recipient) {
        $typeAndId = explode('_', $recipient);
        if ($typeAndId[0] == 0) {
            $userRecipients[] = $typeAndId[1];
        } else {
            $groupRecipients[] = $typeAndId[1];
        }
   }
```
In this case 7 and 5 are user ids, 1 is the group id of an administrator.

## Possible Solution
The code for the directus-sdk-javascript could look like this:
```JAVASCRIPT
   sendMessage(data = requiredParam('data')) {
    	return this._post('messages/rows/', data);
   }
```
I tested the code and it worked for me.
Therefore, the API should mention that POST request to `messages/rows/` is possible and **"recipients"** should be a string with ids in form of **1_id** for group id and **0_id** for user id, separated by a  ",".

I would be happy to receive a quick update for directus-sdk-javascript.
Thanks!

**Comment by Padle** [comment]
This was my first Fork and Pull request. Hope I did it right. Thanks @rijkvanzanten

**Comment by rijkvanzanten** [comment]
Thanks! I saw it :) I’m away from my computer right now, but I’ll publish it to npm first thing tomorrow 👍🏻

**Comment by rijkvanzanten** [comment]
This has been published to npm in version 2.10.0
